### PR TITLE
Update configuring-commit-squashing-for-pull-requests.md

### PR DESCRIPTION
**Pull request by jasonfb**
clarifies how to find the "Pull requests" section (it's on the General tab but you have to scroll down to see it)

### Why:

The existing documentation is confusing

Closes: 

none

### What's being changed (if available, include any code snippets, screenshots, or gifs):
- Clarifies that the "Pull requests" section of the repo settings are on the general tab of the repo settings. 
- adds screenshot of the settings 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@jasonfb Thanks for submitting a PR! I'll get it triaged for review. ⚡ 

However, the screenshot included is not rendering in the staging preview. I'm not sure a screenshot is necessary for this change, though. Perhaps an update to the wording will achieve the goal, but I'll leave that to the reviewer to decide. 

For future reference, check out our guide on [creating screenshots](https://docs.github.com/en/contributing/writing-for-github-docs/creating-screenshots) 👍

File: content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md [hunk 1]
```diff
@@ -20,9 +20,14 @@ shortTitle: Configure commit squashing
...
+1. On the "General" tab (which is selected by default), scroll down to the section marked "Pull Requests".
+![Screenshot 2023-08-31 at 2 16 53 PM](https://github.com/github/docs/assets/59002/fc027e4c-62f5-4abd-8530-050fa24060a3)
+
+
+
+   Select **Allow squash merging**. This allows contributors to merge a pull request by squashing all commits into a single commit. The default commit message presented to contributors when merging is the commit title and message if the pull request contains only 1 commit, or the pull request title and list of commits if the pull request contains 2 or more commits. {% ifversion ghes = 3.6 %} To always use the title of the pull request regardless of the number of commits in the pull request select **Default to PR title for squash merge commits**.{% endif %}
```
Comment (am-stead) [inline content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md:28]:
```suggestion
1. On the "General" settings page (which is selected by default), scroll down to the section marked "Pull Requests". 
1. Under "Pull Requests", select **Allow squash merging**. This allows contributors to merge a pull request by squashing all commits into a single commit. The default commit message presented to contributors when merging is the commit title and message if the pull request contains only 1 commit, or the pull request title and list of commits if the pull request contains 2 or more commits. {% ifversion ghes = 3.6 %} To always use the title of the pull request regardless of the number of commits in the pull request select **Default to PR title for squash merge commits**.{% endif %}
```

File: content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md [hunk 2]
```diff
@@ -20,9 +20,14 @@ shortTitle: Configure commit squashing
...
 {% ifversion default-merge-squash-commit-message %}
-1. Optionally, under **Allow squash merging**, select the dropdown menu, then click the format of the default squash commit message presented to contributors when merging.
+ Optionally, under **Allow squash merging**, select the dropdown menu, then click the format of the default squash commit message presented to contributors when merging.
```
Comment (am-stead) [inline content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md:30]:
```suggestion
1. Optionally, under **Allow squash merging**, select the dropdown menu, then click the format of the default squash commit message presented to contributors when merging.
```

File: content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md (hunk 1 above)
Comment (am-stead) [reply content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md:28]: I don't think the screenshot is strictly necessary in this case (we tend to use them sparingly and I think your proposed clarification to the content is sufficient!). ✨

I'm also suggesting a slight edit to the structure - in general, keeping one action per step.

File: content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md (hunk 2 above)
Comment (am-stead) [reply content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md:30]: Minor edit to align with our style guide.

**Review by am-stead — COMMENTED** [review]
hi @jasonfb, 
Thank you so much for spotting this and for taking the trouble to open a PR for this improvement. I agree, the existing content seems like it could create confusion!
I've proposed some minor edits so that the content aligns more closely with our [style guide](https://docs.github.com/en/contributing/writing-for-github-docs/style-guide), if you could take a look and then we'll get it merged! Thanks for this contribution! ✨

**Review by am-stead — APPROVED** [review]

**Comment by am-stead** [comment]
Hi @jasonfb, 
I'm going to go ahead and merge in your changes. 
Thanks so much for contributing to our docs! ✨

**Pull request merged**